src/common/icons/DocumentRed.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function DocumentRed({ styling }) {
+  return (
+    <svg
+      viewBox="0 0 16 21"
+      xmlns="http://www.w3.org/2000/svg"
+      className={styling}
+    >
+      <path
+        d="M15.856 4.236L11.823.146A.49.49 0 0011.475 0H1.17A1.18 1.18 0 000 1.187V20.5a.493.493 0 00.494.5h.002l14.337-.059h-.001A1.182 1.182 0 0016 19.755V4.589a.506.506 0 00-.144-.353zm-3.891-2.53l2.35 2.383H12.15a.187.187 0 01-.184-.185V1.705zm2.863 18.236l-13.842.056V1.188A.188.188 0 011.171 1h9.808v2.903a1.18 1.18 0 001.17 1.186h2.865v14.665a.186.186 0 01-.186.186zM2.437 7.376c0-.277.22-.5.492-.5h5.907a.492.492 0 01.504.5.495.495 0 01-.504.5H2.929a.49.49 0 01-.348-.147.501.501 0 01-.144-.353zm0-3.035c0-.276.22-.5.492-.5h3.382a.497.497 0 01.483.5.497.497 0 01-.483.5H2.929a.487.487 0 01-.348-.147.501.501 0 01-.144-.353zm10.059 6.07c0 .277-.221.5-.493.5H2.929a.488.488 0 01-.356-.142.504.504 0 010-.715.493.493 0 01.356-.143h9.074c.13 0 .256.053.348.146a.503.503 0 01.145.355zm0 3.036c0 .276-.221.5-.493.5H2.929a.495.495 0 01-.503-.5.492.492 0 01.504-.5h9.073c.13 0 .256.053.348.146a.504.504 0 01.145.354zm0 3.035c0 .276-.221.5-.493.5H2.929a.493.493 0 01-.503-.5.493.493 0 01.504-.5h9.073a.49.49 0 01.348.146.503.503 0 01.145.354z"
+        fill="#9D2E24"
```

Comment:
Let's apply colors to these svg icons using styling so that we can avoid the duplicates for different colors.